geo/src/algorithm/affine_ops.rs

```diff
@@ -312,7 +279,41 @@ impl<T: CoordNum + Neg> AffineTransform<T> {
     }
 }
 
-impl<T: CoordNum + Neg> fmt::Debug for AffineTransform<T> {
+impl<T: CoordNum + Neg<Output = T>> AffineTransform<T> {
```

Comment:
Should we leave this as `Neg` and return `None` if the conversion fails? It makes sense to me.

Reply:
Probably safer, although I'm not sure _how_ it could fail in the first place (`NaN` appearing somewhere?). Have a look at the latest…

Reply:
One more nit :⁠-⁠).